frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/breakdown-card.scss

```diff
@@ -0,0 +1,57 @@
+@import '../../../../../../console-shared/src/styles/skeleton-screen.scss';
+
+.capacity-breakdown-card__available-body {
+  float: right;
+  font-weight: var(--pf-global--FontWeight--normal);
+}
+
+.capacity-breakdown-card__capacity-body {
+  font-size: var(--pf-global--FontSize--sm);
+  margin-bottom: 0;
+}
+
+.capacity-breakdown-card__header-link {
+  padding: var(--pf-global--spacer--xs);
+  margin-right: var(--pf-global--spacer--md);
```

Comment:
nit: properties should be ordered alphabetically

Reply:
There are a lot of places where this needs to be done. We are doing some effort from our side to come up with linter rules. Ex: https://github.com/openshift/console/pull/8131
We can take up ordering of CSS properties in the future.